### PR TITLE
shell/nib: fix minor inconsistency

### PR DESCRIPTION
**Pull request by bergzand**
2 small inconsistencies in the `nib` command. First one is the help output where the optional argument of the show subcommands was displayed as mandatory. Second one is that the `nib prefix add` command takes milliseconds for the lifetimes, while the output of the prefix command displays seconds. This changes the input argument to seconds instead of milliseconds. This way it is also consistent with the `nib route add` subcommand

### CODE REVIEW

File: sys/shell/commands/sc_gnrc_ipv6_nib.c [hunk 1]
```diff
@@ -57,16 +57,16 @@ static void _usage_nib_neigh(char **argv)
...
     printf("       %s %s del <ipv6 addr>\n", argv[0], argv[1]);
-    printf("       %s %s show <ipv6 addr>\n", argv[0], argv[1]);
+    printf("       %s %s show [ipv6 addr]\n", argv[0], argv[1]);
 }
 
 static void _usage_nib_prefix(char **argv)
 {
     printf("usage: %s %s [show|add|del|help]\n", argv[0], argv[1]);
-    printf("       %s %s add <iface> <prefix>[/<prefix_len>] [<valid in ms>] [<pref in ms>]\n",
+    printf("       %s %s add <iface> <prefix>[/<prefix_len>] [<valid in sec>] [<pref in sec>]\n",
```
Comment (miri64) [inline sys/shell/commands/sc_gnrc_ipv6_nib.c]: I don't know about this. The lifetime *has* millisecond precision, so why not use it. Also this would require, to check, that `ltime < (UINT32_MAX / MS_PER_SEC)` which makes the shell command more complicated than it needs to be IMHO. 

I don't see it as an inconsistency either, that's why the times are documented here.

File: sys/shell/commands/sc_gnrc_ipv6_nib.c [hunk 2]
```diff
@@ -57,16 +57,16 @@ static void _usage_nib_neigh(char **argv)
     printf("usage: %s %s [show|add|del|help]\n", argv[0], argv[1]);
     printf("       %s %s add <iface> <ipv6 addr> [<l2 addr>]\n", argv[0], argv[1]);
     printf("       %s %s del <ipv6 addr>\n", argv[0], argv[1]);
-    printf("       %s %s show <ipv6 addr>\n", argv[0], argv[1]);
+    printf("       %s %s show [ipv6 addr]\n", argv[0], argv[1]);
```
Comment (miri64) [inline sys/shell/commands/sc_gnrc_ipv6_nib.c]: This is supposed to be `[iface]`. Would you be so kind and change this as well?

Comment (bergzand) [reply sys/shell/commands/sc_gnrc_ipv6_nib.c]: Ack

File: sys/shell/commands/sc_gnrc_ipv6_nib.c (hunk 1 above)
Comment (bergzand) [reply sys/shell/commands/sc_gnrc_ipv6_nib.c]: > I don't know about this. The lifetime has millisecond precision, so why not use it.

Because IMHO it is overkill for a shell application, when I really require my prefix to have millisecond accuracy I would use a programmatical way to add the prefix as the inaccuracy of the shell is far too large to achieve that millesecond accuracy.
With the millisecond accuracy, the whole process of manually adding a prefix becomes more error prone due to the large numbers that are less readable.

> that ltime < (UINT32_MAX / MS_PER_SEC) which makes the shell command more complicated than it needs to be IMHO.

Fair enough.

> I don't see it as an inconsistency either, that's why the times are documented here.

Okay, let me rephrase that: Why does the `prefix add` command use millisecond for the input and is the `prefix show` using seconds for the output?

File: sys/shell/commands/sc_gnrc_ipv6_nib.c (hunk 2 above)
Comment (bergzand) [reply sys/shell/commands/sc_gnrc_ipv6_nib.c]: Done

File: sys/shell/commands/sc_gnrc_ipv6_nib.c (hunk 1 above)
Comment (miri64) [reply sys/shell/commands/sc_gnrc_ipv6_nib.c]: Mhhhhh... I think I am convinced.

**Comment by miri64** [comment]
(also, please add the check we talked about)

**Comment by bergzand** [comment]
> (also, please add the check we talked about)

I've added the check. For now I've decided on the same approach as with the RPL and PIO lifetimes and truncate values that are too large to `UINT32_MAX - 1`. I have no strong opinion on whether to truncate or return an error to a user, so if you rather have a `printf()` with a message and a `return 1` I can replace it with that.

**Comment by bergzand** [comment]
squashed
